modules/magniteAnalyticsAdapter.js

```diff
@@ -123,11 +123,15 @@ const addEventToQueue = (event, auctionId, eventName) => {
   }
 }
 
-const sendEvent = payload => {
+const sendEvent = (payload, isUnload) => {
   const event = {
     ...getTopLevelDetails(),
     ...payload
   }
+  if (isUnload) {
+    navigator.sendBeacon(endpoint, JSON.stringify(event));
+    return;
+  }
```

Comment:
navigator.sendBeacon is quicker / more reliable during `beforeunload` than prebid's AJAX library